pkg/awx/job.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file contains the implementation of the job template type.
+
+package awx
+
+type Job struct {
+	id     int
+	status string
+}
+
+func (j *Job) Id() int {
+	return j.id
+}
+
+func (j *Job) Status() string {
```

Comment:
Are the status known strings? If they are then I think it would be useful to have constants for them, so that users know what to expect:

```go
type JobStatus string

const (
  JobStatusRunning JobStatus = "running"
  JobStatusFailed JobStatus = "failed"
  ...
)
```
)